cmd/skaffold/app/cmd/cmd.go

```diff
@@ -58,6 +58,7 @@ func NewSkaffoldCommand(out, err io.Writer) *cobra.Command {
 }
 
 func AddRunDevFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&opts.CustomTag, "tag", "t", "", "The optional custom tag to use for images which overrides the current Tagger configuration")
```

Comment:
These flags are shared between run and dev, maybe it makes sense just for `skaffold run`

Reply:
good call - lemme move it to run only flags...